### ISSUE TITLE
Add Docker feature

### ISSUE THREAD

**Issue by salamisodikiolawale** [jmprathab/MyHome-Web#221]
## Docker feature
Hi, me and my friend @MargotRasamy wanted to contribute to the project by adding docker functionality with docker-compose.
This functionality will make the installation of the project and its deployment easier.

- Add Dockerfile 
- Add docker-compose file to create MyHome-web service with container

## 🤚 Do you want to develop this feature yourself?

- [ x ] Yes
- [ ] No

**Comment by siddhantSharma007** [comment]
yes, give me the access of repo and i can add feature with dockerfile and docker-compose also.
